net/netclient/Makefile

```diff
@@ -0,0 +1,56 @@
+# Copyright 2024 Stan Grishin (stangri@melmac.ca)
+# This is free software, licensed under the Apache-2.0 License.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=netclient
+PKG_VERSION:=0.22.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/gravitl/netclient/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=010e7ee72f8cf8ae4cc91b19010b85875d75e70faabd248c992af2487629211d
+
+PKG_MAINTAINER:=Stan Grishin <stangri@melmac.ca>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE.txt
+
+PKG_BUILD_DEPENDS:=golang/host
+PKG_BUILD_PARALLEL:=1
+PKG_BUILD_FLAGS:=no-mips16
+
+GO_PKG:=github.com/gravitl/netclient
+GO_PKG_BUILD_PKG:=github.com/gravitl/netclient
+GO_PKG_LDFLAGS_X:=main.version=v$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../../lang/golang/golang-package.mk
+
+define Package/netclient
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=netclient
+  URL:=https://docs.openwrt.melmac.net/netclient/
+  DEPENDS:=$(GO_ARCH_DEPENDS) +wireguard-tools
+endef
+
+define Package/netclient/description
+This is the client for Netmaker networks. To learn more about Netmaker, see URL below:
+https://github.com/gravitl/netmaker
+endef
+
+define Package/netclient/install
+	$(call GoPackage/Package/Install/Bin,$(PKG_INSTALL_DIR))
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/netclient.init $(1)/etc/init.d/netclient
+	$(SED) "s|^\(readonly PKG_VERSION\).*|\1='$(PKG_VERSION)-$(PKG_RELEASE)'|" $(1)/etc/init.d/netclient
+	$(INSTALL_DIR) $(1)/usr/sbin
+	$(INSTALL_BIN) $(PKG_INSTALL_DIR)/usr/bin/netclient $(1)/usr/sbin/
+	$(INSTALL_DIR) $(1)/lib/upgrade/keep.d
+	$(INSTALL_DATA) ./files/netclient.upgrade $(1)/lib/upgrade/keep.d/netclient
+	$(INSTALL_DIR) $(1)/usr/share/doc/netclient
+	$(INSTALL_DATA) $(PKG_BUILD_DIR)/$(PKG_LICENSE_FILES) $(1)/usr/share/doc/netclient/$(PKG_LICENSE_FILES)
```

Comment:
Let me think. Why? I dont think that majority of packages are doing this in the way like you do. Thats the PKG_LICENSE and PKG_LICENSE_FILES are for. We dont need to put it in the package even though, we might, but dont forget that OpenWrt tries to be minimalize as possible. So it is not huge.

Reply:
https://www.apache.org/licenses/LICENSE-2.0 -- check the 4a. I believe this contradicts your statement:
> We dont need to put it in the package

Reply:
Again, look how other packages are done it. Here in this community maintained repository and as well in the main repo. How many package are doing it? How PKG_LICENSE_FILES handles it? How many packages are shipping its LICENSE files?

Reply:
I can raise an issue with at the upstream, but unless they explicitly grant me a permission to not include a license file with the binary, I'd be very uncomfortable to drop it. Also, the main binary is ~ 6Mb, so size-wise including a license file is not that big of a deal.

Reply:
The `PKG_LICENSE_FILES` is already properly set in the Makefile. Can you educate me on what happens to the files listed there, so that I don't have to install the license file on the system?

Reply:
Its interesting that after 2 years, we are getting to the same conversation and you are still trying to slip it here. 🤔 
https://github.com/openwrt/packages/pull/14548#discussion_r563584143

Reply:
It's also interesting that after 2 years, nobody addressed my question on that and the discussion has been resolved by another party.